package.json

```diff
@@ -57,15 +57,18 @@
     "tar": "^4.4.19",
     "tmp": "^0.0.33",
     "yargs": "^16.0.3",
-    "yn": "^2.0.0"
+    "yn": "^2.0.0",
+    "joi": "^13.6.0"
   },
   "resolutions": {
     "fstream": ">=1.0.12",
     "lodash": ">=4.17.21",
     "@jest/types": ">=27.4.2",
     "@types/jest": ">=27.4.0",
     "@types/puppeteer": ">=5.4.4",
-    "jest-circus": ">=27.4.6"
+    "jest-circus": ">=27.4.6",
+    "@types/joi": "^13.4.5",
+    "joi": "^13.6.0"
```

Comment:
This could potentially cause problems in other places that use a different version of jest. I'm not sure if modules are affected by this